### ISSUE TITLE
Add OnResumeFragments event for FragmentActivity

### ISSUE THREAD

**Issue by ndori** [trello/RxLifecycle#144]
Hi, 
This is an important event, because only after it we can use the fragmentManager without getting an exception of execute pending transactions while onSaveInstanceState is called.

I personally used it to postpone any fragment transaction to a point where it can be run without forgetting about it but I guess there are other scenarios as well.

In my code I've changed ActivityEvent.RESUME to be run after this call instead of the regular onResume but that because I didn't want to change the ActivityEvent enum. so it might be a better change over there.

But I guess this event is only applicable to FragmentActivity so a different approach might be needed 

Tnx, Nitzan

**Comment by dlew** [comment]
How are you using RxLifecycle in this context? Are you using the `Observable<ActivityEvent>` to listen for `ActivityEvent.RESUME`, and then posting fragment transactions in there? Or are you just wanting to bind to a different event?

**Comment by ndori** [comment]
The first one, I'm using the lifecycle() method and filters the RESUME, then using either delay with lifecycle() as the delay observable or combineWith, depending on exactly what I want to achieve

**Comment by dlew** [comment]
That's not really the general use case for RxLifecycle. That's why we created [navi](https://github.com/trello/navi/) which has a lot more callbacks from the lifecycle.

That also doesn't have `onResumeFragments()` but that could easily be added there.

**Comment by ndori** [comment]
o.k I will check it out, so as I understand it, you only want RxLifecycle to handle the unsubscription of subscribers (which performs through onComplete..)  right?

I actually had another use case which is more appropriate to unsubscription , I will post it tomorrow.

Tnx for all the work, It is much appreciated
